src/context.rs

```diff
@@ -102,6 +141,26 @@ impl Context {
         out_point
     }
 
+    pub fn deploy_cell_by_name(&mut self, filename: &str) -> OutPoint {
+        let path = self.contracts_dir.join(filename);
+        let data = std::fs::read(path).expect("read local file");
+
+        #[cfg(feature = "simulator")]
+        {
+            let native_path = self.contracts_dir.join(format!(
+                "lib{}_dbg.{}",
+                filename.replace("-", "_"),
```

Comment:
`-` is a valid character in path.

Reply:
Normally, Onchain-script compiles to `-`, while cdylib compiles to `_`, So in this replacement is needed.
e.g.  Onchain-script name : aaa-bbb. Compiled on Linux: libaaa_bbb_dbg.so.